packages/dev/core/src/Compat/compatibilityOptions.ts

```diff
@@ -1,9 +1,6 @@
+/* Options used to control default behaviors regarding compatibility support */
+
 /**
- * Options used to control default behaviors regarding compatibility support
+ * Defines if the system should use OpenGL convention for UVs when creating geometry or loading .babylon files (false by default)
  */
-export class CompatibilityOptions {

```

Comment:
unfortunately this is a breaking change.

Reply:
THAT BEING SAID, @sebavan this is purely internal right?

Reply:
It is exported so hard to tell if used publicly :-( It is a breaking change in its current form and wondering what does it bring pragmatically ?

Reply:
This is more of a cleanup thing. If you like, I can keep the class but mark it as deprecated?

Reply:
Let s see what @RaananW thinks as well

Reply:
@deltakosh @sebavan,

I'm going to include an object for backward compatibility. It is marked as deprecated. (https://github.com/BabylonJS/Babylon.js/pull/15376/commits/57728a71a7dede24eccae72d220c440666b94e0e)